sdks/python/src/opik/url_helpers.py

```diff
@@ -1,8 +1,24 @@
 import opik.config
+import opik.api_objects.opik_client
 
 
 def get_ui_url() -> str:
     config = opik.config.OpikConfig()
     opik_url_override = config.url_override
 
     return opik_url_override.rstrip("/api")
+
+
+def get_experiment_url(dataset_name: str, experiment_id: str) -> str:
+    client = opik.api_objects.opik_client.get_client_cached()
+
+    # Get dataset id from name
+    dataset = client._rest_client.datasets.get_dataset_by_identifier(
+        dataset_name=dataset_name
+    )
+    dataset_id = dataset.id
+
+    config = opik.config.OpikConfig()
+    ui_url = get_ui_url()
```

Comment:
When this method is called, OpikConfig will be reinitialized again, which can be considered a resource-intensive operation as it involves reading environment variables, config file, etc.

Reply:
@japdubengsub The `evaluate` function is already very resource intensive and takes at least 15 seconds to run, don't think this will be an issue. Is there another way to get the configuration that is already read ? If not, we can keep as is